src/Notifier.ts

```diff
@@ -56,7 +57,7 @@ This is useful when the content body contains fallback text that would explain t
 type of tile.
 */
 const typehandlers = {
```

Comment:
can we rename this to msgtypehandlers? otherwise it implies eventtype which it isn't